docs/tutorial/part-six/index.md

```diff
@@ -169,11 +154,11 @@ export const query = graphql`
 `
 ```
 
-Now the frontpage should look like:
+હવે આગળનું પાનું આના જેવું દેખાવું જોઈએ:
```

Comment:
Just Translate frontpage in gujarati. Don't transalate it like `આગળનું પાનું`. It will change the meaning of the frontpage.